src/test/java/com/teradata/tpcds/CallCenterGeneratorTest.java

```diff
@@ -24,6 +24,17 @@
 {
     private static final Session TEST_SESSION = getDefaultSession().withTable(CALL_CENTER);
 
+    // A scale of 0.1 is not part of the spec. Support for it was added because of Presto. Since
```

Comment:
I would say more generally that it was added in order to generate data sets smaller than 1gb for testing purposes.